base/linalg/triangular.jl

```diff
@@ -976,18 +976,110 @@ for (f, g) in ((:/, :A_rdiv_B!), (:A_rdiv_Bc, :A_rdiv_Bc!), (:A_rdiv_Bt, :A_rdiv
     end
 end
 
+function ^(A::UpperTriangular, p::Integer)
```

Comment:
Agreed, but I'd rather prefer not to do it with this PR, because it would break the consistency with the notation of `sqrtm`. I'll put it in my TODO list.